lib/collections/schemas/accounts.js

```diff
@@ -131,93 +136,87 @@ registerSchema("Email", Email);
  * @property {Date} updatedAt optional
  */
 export const Accounts = new SimpleSchema({
-  userId: {
+  "userId": {
     type: String,
     regEx: SimpleSchema.RegEx.Id,
     index: 1,
     label: "Accounts ShopId"
   },
-  sessions: {
-    type: [String],
+  "sessions": {
+    type: Array,
     optional: true,
     index: 1
   },
-  shopId: {
+  "sessions.$": {
+    type: String
+  },
+  "shopId": {
     type: String,
     autoValue: shopIdAutoValue,
     regEx: SimpleSchema.RegEx.Id,
     index: 1
   },
-  name: {
+  "name": {
     type: String,
     optional: true
   },
-  username: {
+  "username": {
     type: String,
     optional: true
   },
-  emails: {
-    type: [Email],
+  "emails": {
+    type: Array,
     optional: true
   },
-  acceptsMarketing: {
+  "emails.$": {
+    type: Email
+  },
+  "acceptsMarketing": {
     type: Boolean,
     defaultValue: false,
     optional: true
   },
-  state: {
+  "state": {
     type: String,
     defaultValue: "new",
     optional: true
   },
-  taxSettings: {
+  "taxSettings": {
     type: TaxSettings,
     optional: true
   },
-  note: {
+  "note": {
     type: String,
     optional: true
   },
-  profile: {
+  "profile": {
     type: Profile,
     optional: true
   },
-  groups: {
-    type: [String], // groupIds that user belongs to
+  "groups": {
+    type: Array, // groupIds that user belongs to
     optional: true,
     defaultValue: []
   },
-  metafields: {
-    type: [Metafield],
+  "groups.$": {
+    type: String
+  },
+  "metafields": {
+    type: Array,
     optional: true
   },
-  createdAt: {
+  "metafields.$": {
+    type: Metafield
+  },
+  "createdAt": {
     type: Date,
-    autoValue() {
-      if (this.isInsert) {
-        return new Date;
-      } else if (this.isUpsert) {
-        return {
-          $setOnInsert: new Date
-        };
-      }
-    }
-  },
-  updatedAt: {
+    autoValue: createdAtAutoValue
```

Comment:
Tried to update all the createdAt/updatedAt to use the same functions, unless there seemed to be a good reason for it to be done differently.